app/views/shared/_footer.html.erb

```diff
@@ -12,11 +12,10 @@
     <div class="footer-links-container">
       <%= link_to "About", about_path %>
       <%= link_to "FAQ", faq_path %>
-      <%= link_to "Blog", "http://1000hours.io" %>
+      <%= link_to "Blog", "https://medium.com/the-odin-project" %>
```

Comment:
Nitpick. Can you add a method `medium_blog_url` in `ApplicationHelper` and use that here?

```
def medium_blog_url
  'https://medium.com/the-odin-project'
end
```